doc/articles/get-started-vs.md

```diff
@@ -1,4 +1,4 @@
-# Get Started on Visual Studio 
+# Get Started on Visual Studio 2019
```

Comment:
Should we reconsider changing the file name to reflect the VS version number?
I would expect the "get-started-vs,md" to be for the latest stable version of VS